lib/platform/aws-lambda/prices.js

```diff
@@ -1,22 +1,29 @@
 // Compute pricing for AWS Lambda
 // GB/second divided by 10e10
+// https://aws.amazon.com/lambda/pricing/
 module.exports = {
   base: {
-    x86: 1666670
```

Comment:
I think this needed to be adjusted:
Price for us-east-1 ([aws](https://aws.amazon.com/lambda/pricing/)):
`$0.0000166667` for every GB-second
`1666670 / 10 ^ 10 = 0.000166667`
If im not missing anything we were 'overcharging'

Reply:
10e10 = 10 * 10^10